docs/islandora/rdf-mapping.md

```diff
@@ -1,39 +1,132 @@
+
+
+### REST API
+The pattern of using `?_format=` to get a different representation of content is provided by the RESTful Web Services (rest) module. It allows other services to interact with Drupal entities through HTTP requests (`GET`, `POST`, `PATCH`, and `DELETE`). Which operations are allowed, and with what formats (such as `xml`, `json`, and `jsonld`) is configured at `admin/config/services/rest/`. Note that only `jsonld` uses RDF mappings; the `json` and `xml` formats expose a structured object based on how Drupal sees the entity. Access to these alternate formats throught the REST API corresponds to permissions on the entity, so anyone with `access content` permission can view the JSON-LD version of that content. This is new as of[ Drupal 8.2](https://www.drupal.org/docs/8/api/restful-web-services-api/restful-web-services-api-overview#practical).
+
+For more information on interacting with Drupal entities via REST requests, see [An Introduction to RESTful Web Services in Drupal 8](https://drupalize.me/blog/201401/introduction-restful-web-services-drupal-8).
+
+## JSON-LD Module
+
+Using the RDF mapping configurations provided by the RDF module, the JSON-LD Module exposes the RDF-mapped entity in JSON-LD, through the REST API, at `node/[nid]?_format=jsonld` (for nodes; for media and terms, at `media/[mid]?_format=jsonld` and `taxonomy/term/[tid]?_format=jsonld`).
+
+- The JSON-LD module will only work with mappings that include a value under `types` (which maps to `rdf:type` - see below, under Structure of an RDF Mapping).
+- The JSON-LD module provides a hook so other modules can alter the entity before it gets mapped. The `islandora` module uses this hook to trigger any "Map URI to Predicate" and "Alter JSON-LD Type" reactions that are configured in Contexts. `islandora_defaults` provides the two Contexts - "All Media" and "Content" - that configure these to occur on Media and Repository Item nodes.
+- The JSON-LD module adds RDF datatypes to the RDF values, and includes a mapping of Drupal field types to RDF datatypes.
+- The JSON-LD module provides a hook to alter its Drupal field type to RDF datatype mapping.
+- The JSON-LD module has a configuration option that can cause the `?_format=jsonld` to be part of, or not part of, the URIs of Drupal objects. On an out-of-the-box claw-playbook, this string is stripped, but by default on a fresh install of the jsonld module, it is not.
+
+
+## RDF Mappings
+In an out-of-the-box claw-playbook, the RDF mappings that exist were loaded from config files, and correspond to the rdf.mapping.[...].yml files located in:
+
+- `[drupal modules directory]/islandora/modules/islandora_core_feature/config/install/` (media and taxonomy terms)
+- `[drupal modules directory]/islandora_defaults/config/install/` (`repository_item` and the `islandora_access` vocabulary)
+- `[drupal web root]/core/profiles/standard/config/install/` (articles, pages, comments, and tags).
+
```

Comment:
Also, `[drupal modules directory]/controlled_access_terms/modules/controlled_access_terms_defaults/config/install/` (for controlled access terms vocabularies).

Reply:
thanks!